src/sap.ui.core/src/sap/ui/model/CompositeBinding.js

```diff
@@ -53,14 +57,12 @@ sap.ui.define(['jquery.sap.global', './BindingMode', './ChangeReason', './Proper
 	};
 	
 	CompositeBinding.prototype.isResolved = function() {
-		var bResolved = false;
 		jQuery.each(this.aBindings, function(i, oBinding) {
-			bResolved = oBinding.isResolved();
-			if (!bResolved) {
+			if (!oBinding.isResolved()) {
 				return false;
 			}
 		});
-		return bResolved;
+		return true;
```

Comment:
Thanks for pointing this out - you are totally right, I will revert it.
I agree on the usage of Array.prototype.forEach() but only saw jQuery.each used and thought this is the internal way to go.